lib/foreman_tasks_core/runner/base.rb

```diff
@@ -42,7 +42,7 @@ def close
       end
 
       def publish_data(data, type)
-        @continuous_output.add_output(data, type)
+        @continuous_output.add_output(data.force_encoding('UTF-8'), type)
```

Comment:
shouldn't we override the `publish_method` in rex to call `super(data.force_endocing('UTF-8'), type)` there